exchange/utils.go

```diff
@@ -25,19 +25,19 @@ import (
 )
 
 var integrationTypeMap = map[metrics.RequestType]config.IntegrationType{
```

Comment:
Let's use the proper naming here: `var channelTypeMap = map[metrics.RequestType]config.ChannelType{`